distribution/kubernetes/vector-aggregator/README.md

```diff
@@ -0,0 +1,7 @@
+The kubernetes manifests found in this directory have been automatically generated
+from the [helm chart `vector/vector`](https://github.com/vectordotdev/helm-charts/tree/master/charts/vector)
+version 0.3.0 with the following `values.yaml`:
+
+```yaml
+
+```
```

Comment:
The empty content in here - is that expected?

Reply:
It's entirely default values so yes, could perhaps make it not render at all or something more elegant

Reply:
It's technically correct_